layers/+tools/geben/packages.el

```diff
@@ -0,0 +1,72 @@
+;;; packages.el --- geben layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Franz Luther Neulist Carroll <franzneulistcarroll@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defconst geben-packages '(geben))
+
+
+(defun geben/init-geben ()
+  (use-package geben
+    :init
+    (progn
+      (spacemacs/set-leader-keys
+        "Ga" 'geben
+        "Gr" 'geben-run
+        "Gx" 'geben-stop
+        "Ge" 'geben-end
+        "Gp" 'geben-proxy
+        "Gn" 'geben-proxy-end
+        "Gw" 'geben-where
+        "Gq" 'geben-quit-window
+        "Gv" 'geben-set-redirect
+        "Gh" 'geben-mode-help
+        "Gm" 'geben-toggle-pause-at-entry-line-flag
+        "Gj" 'geben-eval-expression
+        "Gk" 'geben-eval-current-word
+        "Gi" 'geben-run-to-cursor
+        "Gg" 'geben-full-frame-mode
+        "Go" 'geben-open-file
+        "Gf" 'geben-find-file
+
+        "Gcd" 'geben-display-context
+        "Gcm" 'geben-context-mode
+        "Gch" 'geben-context-mode-help
+        "Gcr" 'geben-context-mode-refresh
+
+        "Gtt" 'geben-show-backtrace
+        "Gtm" 'geben-backtrace-mode
+        "Gth" 'geben-backtrace-mode-help
+        "Gtg" 'geben-backtrace-mode-goto
+        "Gth" 'geben-backtrace-mode-mouse-goto
+        "Gtc" 'geben-backtrace-mode-mode-context
+
+        "Gsi" 'geben-step-into
+        "Gso" 'geben-step-over
+        "Gsu" 'geben-step-out
+        "Gsa" 'geben-step-again
```

Comment:
If the user is likely to use these commands several times consecutively, we should consider putting them in a transient state (see `spacemacs|define-transient-state`). For example, this way the user can type `SPC G s . o o o` (or similar) instead of `SPC g s o SPC g s o SPC g s o` to step-over multiple times.

I don't actually know anything about Geven, other from looking at the repo, so please excuse me if I've missed something relevant.